sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -63,6 +63,8 @@ export enum AppRoutes {
   Reports = "reports",
   ApplicationExceptionsApproval = "application-exceptions/:exceptionId",
   Exceptions = "exceptions",
+  OfferingChangeRequests = "institution/offering/change-requests",
+  ViewOfferingChangeRequest = "institution/offering/view-change-request/:offeringId/program/:programId",
```

Comment:
I believe that this one ca follow the pattern from `OfferingChangeRequests` like below:
````typescript
ViewOfferingChangeRequest = "institution/offering/change-requests/:offeringId/program/:programId"
```